libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -215,8 +215,16 @@ string ABIFunctions::cleanupFunction(Type const& _type, bool _revertOnFailure)
 			templ("body", "cleaned := iszero(iszero(value))");
 			break;
 		case Type::Category::FixedPoint:
-			solUnimplemented("Fixed point types not implemented.");
+		{
```

Comment:
This seems to be identical to the respective integer type. In view of the optimizer, it might be bettor to just use
`return cleanupFunction(type.integerType())`.

Reply:
Or something like
```
templ("body", "cleaned := " + cleanupFunction(type.integerType() + "(value)");
```